services/motion/motion.go

```diff
@@ -96,16 +97,21 @@ type ListPlanStatusesReq struct {
 	Extra           map[string]interface{}
 }
 
-// Plan represents a motion plan.
-type Plan struct {
+// PlanStep represents a single step of the plan
+// Describes the pose each resource described by the plan
+// should move to at that step.
+type PlanStep map[resource.Name]spatialmath.Pose
```

Comment:
Don't we have another type with this name in `motionplan`?

Reply:
Oops I meant to remove this to have the only version exist in `motionplan`